pkgs/development/python-modules/laszip/default.nix

```diff
@@ -0,0 +1,61 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, scikit-build-core
+, distlib
+, pytestCheckHook
+, pyproject-metadata
+, pathspec
+, pybind11
+, cmake
+, LASzip
+}:
+
+buildPythonPackage rec {
+  pname = "laszip-python";
+  version = "0.2.1";
+
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    rev = version;
+    repo = pname;
+    owner = "tmontaigu";
+    sha256 = "sha256-ujKoUm2Btu25T7ZrSGqjRc3NR1qqsQU8OwHQDSx8grY=";
+  };
+
+  propagatedBuildInputs = [
+    scikit-build-core
+    distlib
+    pyproject-metadata
+    pathspec
+  ];
+
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = [
+    pybind11
+    LASzip
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  preBuild = ''
+    cd ..
+  '';
+
+  # There are no tests, and pytest causes stdenv setup script to crash in the
+  # case of no tests here
+  doCheck = false;
+
+  pythonImportsCheck = [ "laszip" ];
+
+  meta = with lib; {
+    description = "Unofficial bindings between Python and LASzip made using pybind11";
+    homepage = "https://github.com/tmontaigu/laszip-python";
```

Comment:
license is missing

Reply:
@SuperSandro2000 the upstream does not contain a license, so I'm not sure what it is.

Reply:
No license means all rights reserved, or just, unfree.

Reply:
https://github.com/tmontaigu/laszip-python/issues/6